docs/flink-connector/flink-connector.md

```diff
@@ -23,7 +23,7 @@ This capability allows users to perform federation queries, accessing data from
 
 ## How to use it
 
-1. [Build](../how-to-build.md) or [download](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector) the Gravitino flink connector jar, and place it to the classpath of Flink.
+1. [Build](../how-to-build.md) or [download](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector) the Gravitino flink connector runtime jar, and place it to the classpath of Flink.
```

Comment:
@yuqi please help to verify that the flink runtime jar link will be `https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector-runtime` ?

Reply:
Let me check it.

Reply:
There is no artifact named `gravitino-flink-connector-runtime`, only `gravitino-flink-connector-runtime-1.18.0_2.12` and `gravitino-flink_2.12`.

Reply:
so the download link should be `https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector-runtime-1.18.0_2.12`?

Reply:
we should use `1.18` not `1.18.0` here?  and link should be `https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector-runtime-1.18` like spark not include scala versions.
```
val flinkMajorVersion: String = flink.substringBeforeLast(".")
```

Reply:
@FANNG1 But I saw the spark has the scala version(_2.12). 
![image](https://github.com/user-attachments/assets/83070cd6-5a8e-4701-a2a3-835459d00951)

Reply:
I mean the runtime jar download link doesn't contain scala version

Reply:
https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-spark-connector-runtime-3.4